### PR TITLE
NavbarMobile integration

### PR DESCRIPTION
**Pull request by antonel-iwnl**
sopy have fun ;)

### CODE REVIEW

File: src/components/navbar/mobile/parts/authPopupM/AuthPopupM.tsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
+import GithubLoginButton from '@src/components/auth/old/social-buttons/GithubLoginButton';
+import GoogleLoginButton from '@src/components/auth/old/social-buttons/GoogleLoginButton';
+import {
+  handleGitHubLogin,
+  handleGoogleLogin,
+} from '@src/components/auth/old/socialAuth';
+import dropclose from '@assets/cross.svg';
+import React from 'react';
+
```
Comment (sopyb) [inline src/components/navbar/mobile/parts/authPopupM/AuthPopupM.tsx:9]: Useless component use
setBasePopup('get-started');
from @components/shared/stores/store-base-popups

File: src/components/navbar/mobile/parts/buttons/ButtonsM.tsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+const LoggedButtonsM = ({ isLogged }: { isLogged: boolean }) => {
+  const [authPopup, setAuthPopup] = useState(false);
+
+  const handleAuthClick = () => {
+    setAuthPopup((prev) => !prev);
+  };
+  return (
+    <div className='flex flex-col gap-4 mt-12'>
+      {!isLogged && (
+        <div className='flex flex-row'>
```
Comment (sopyb) [inline src/components/navbar/mobile/parts/buttons/ButtonsM.tsx:15]: Use an object and map them to the different buttons in the side panel

File: src/components/navbar/mobile/parts/navmenu/SlideMenu.tsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+import userStatus from '@store/user/user-status';
+import { useStore } from '@nanostores/react';
+import ButtonsManagerM from '../buttons/ButtonsManagerM';
+
+const SlideMenu = ({ isOpen }: { isOpen: boolean }) => {
+  const { isLogged } = useStore(userStatus);
+
+  return (
+    <motion.div
+      className='fixed top-0 right-0 h-full w-48 bg-[#182852] z-[200]'
```
Comment (sopyb) [inline src/components/navbar/mobile/parts/navmenu/SlideMenu.tsx:12]: Height should be relative to device size maybe w-5/12 the hit target is too small on some phones

File: src/components/navbar/mobile/parts/profileDataM/ProfilePicAndNameNavM.tsx [hunk 4]
```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { useStore } from '@nanostores/react';
+import { storeProfileData } from '@src/components/profile/stores/store-profile-data';
+
+const ProfilePicAndNameNavM = () => {
+  const { data, loading } = useStore(storeProfileData);
```
Comment (sopyb) [inline src/components/navbar/mobile/parts/profileDataM/ProfilePicAndNameNavM.tsx:6]: Broken - throws an error when you're logged in killing the whole navbar

File: src/components/navbar/mobile/parts/searchM/SearchRoadmapM.tsx [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+        width: '0px',
+        paddingLeft: '0px',
+        paddingRight: '0px',
+        paddingTop: '0px',
+        paddingBottom: '0px',
+      }); // Collapse the input
+    } else {
+      controls.start({
+        width: '250px',
+        paddingLeft: '8px',
```
Comment (sopyb) [inline src/components/navbar/mobile/parts/searchM/SearchRoadmapM.tsx:38]: width should be screen width - the size of the other components see [calc](https://developer.mozilla.org/en-US/docs/Web/CSS/calc) this makes the search small on some devices and on other makes it not fit the screen

File: src/components/navbar/mobile/parts/navmenu/SlideMenu.tsx (hunk 3 above)
Comment (sopyb) [reply src/components/navbar/mobile/parts/navmenu/SlideMenu.tsx:12]: Also color should be put in Tailwind.css and refered to by name

File: src/components/navbar/mobile/parts/profileDataM/ProfilePicAndNameNavM.tsx (hunk 4 above)
Comment (sopyb) [inline src/components/navbar/mobile/parts/profileDataM/ProfilePicAndNameNavM.tsx:6]: Use storeLoggedUser here storeProfileData is only defined on /profile

Comment (sopyb) [reply src/components/navbar/mobile/parts/profileDataM/ProfilePicAndNameNavM.tsx:6]: also when using the right store the sidepanel is transparent when open
image

File: src/layouts/NormalLayout.astro [hunk 6]
```diff
@@ -4,6 +4,8 @@ import '@fontsource/kanit';
 import '@fontsource/oxygen';
 import {PageMeta} from '../types/misc/PageMeta';
 
+const { id } = Astro.params
+
```
Comment (sopyb) [inline src/layouts/NormalLayout.astro:8]: ???? - no params for this file

Comment (antonel-iwnl) [reply src/layouts/NormalLayout.astro:8]: făceam debugging la id ignoră:)))

**Pull request merged**